scripts/apps/authoring/widgets/widgets.ts

```diff
@@ -377,7 +378,11 @@ function WidgetsManagerCtrl(
             $scope.active.afterClose($scope);
         }
 
-        $scope.active = null;
+        if ($scope.pinnedWidget != null) {
```

Comment:
shouldn't it close in either case - whether pinned or not?

Reply:
No logic is correct, such that if you have a pinned widget but then open another one on top of it when you close this "unpinned"/"overlayed" one, the pinned is back to active. If that's not the case and you're just closing a widget then it'd get closed as normally.